delta-proto/src/main/java/io/cdap/delta/proto/FullTableAssessment.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.delta.proto;
+
+import io.cdap.delta.api.assessment.Problem;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Full table assessment using assessments from both the source and target.
+ */
+public class FullTableAssessment {
```

Comment:
I feel `FullAssessment` is a better name for this since this contains information about both tables. On a first glance, I think this class is still about one table, but provide more information than `TableAssessment`

Reply:
It's still about one table, though it can contain transformed columns due to schema normalization (and in the future, actual user specified transforms). FullAssessment makes me think of an assessment for the entire pipeline. I think I'll rename this to TableAssessmentResponse, since it's only used as the response for the table assess endpoint.